book/README.md

```diff
@@ -1,48 +1,53 @@
-- Publishing: [`Github Pages`](https://pages.github.com/) hosted at <https://insightsengineering.github.io/tlg-catalog>
+See the full list of available TLGs in [Index page](tlg-index.qmd).
 
-## Development
+The source code of each article can be inspected by clicking on the "Source Code" button. Use copy-paste to get the code and run it in your environment.
+
+![artice code copy](assets/img/article-code-copy.gif)
+
+Source code of an individual output can also be copied.
+
+![chunk code copy](assets/img/chunk-code-copy.gif)
+
+For running the code in your environment, you might want to check reproduciblity information which allows you to install the required packages and run the code.
 
-This repository is based on the [quarto websites](https://quarto.org/docs/websites/) framework.
+![download lockfile](assets/img/article-lock-download.gif)
 
-As a pre-requisite, you need to [install quarto](https://quarto.org/docs/get-started/) and also [install and configure staged.dependencies](https://github.com/openpharma/staged.dependencies#usage).
+## License
+
+This catalog as well as code examples are licensed under the Apache License, Version 2.0 - see the [LICENSE](LICENSE) file for details.
+
+## Contributing
+
+We welcome contributions to this catalog. Please refer to the [Contributing](CONTRIBUTING.md) guide for more information.
+Use `giscus` to share your feedback, ideas, ask questions, or report issues.
+
+## Development
 
-You can install dependencies simply by running the following in an R session:
+This website is built using [Quarto](https://quarto.org/) and hosted on [Github Pages](https://pages.github.com/). This website is rebuilt and republished every time a change is pushed to the repository as well as on a daily basis as a part of the CI/CD process.
 
-``` r
-# Assuming:
-# 1. Your current working directory is the same as the project directory
-# 2. You have set up your GITHUB_PAT environment variable
-options(
-  staged.dependencies.token_mapping = c(
-    "https://github.com" = "GITHUB_PAT"
-  )
-)
-library(staged.dependencies)
-x <- dependency_table(project = ".", verbose = 1)
-install_deps(x, verbose = 1, install_project = FALSE)
-```
+The catalog is being rendered using "Stable" and "Development" profiles.
+The main difference between them is the package versions used when generating the outputs.
+The "Stable" profile uses the latest released versions of all packages, whereas the "Development" profile uses the latest development versions of the NEST packages.
+This means that the same R code (e.g. `foo::bar()`) would be run using both the latest released and development package versions (e.g. `foo@v1.2.3` and `foo@v1.2.3.9123` respectively).
+If there are significant package API changes between released and development versions, please consider [conditional content](https://quarto.org/docs/authoring/conditional.html) (checking `QUARTO_PROFILE` environment variable) and/or `if` statements on respective package version.
```

Comment:
The golden deploy in the stable deploy using released versions. The devel deploy is more of a help to fail fast before development package becomes released. Therefore I would disable / discourage (and not mention) such limitation.
In your example case - an error will be raised during deployment. It's then assumed that this will be picked by development team and resolved promptly. Internally, we have enabled notifications to the integration tests chat room.

Reply:
So if I contribute, who is responsible for my article running on devel? Is it the NEST team? Am I responsible solely for it running on stable at the moment of contribution?

Reply:
Even though the golden one is the stable deploy, at the moment of contribution, your article would be tested on both envirs. AFAIK that's the requirement for merging PR. Starting from then, the NEST dev team(s) would be responsible for upkeep that article for both devel and stable.

Reply:
Understood. My original suggestion stands.